docs/portal/registry-users/editing-users.md

```diff
@@ -0,0 +1,11 @@
+# Editing Registry Users
+
+Editing a registry user allows you to see basic information about the user along with the projects associated with the user. Editing also allows you to edit the repositories associated with the user. Buttons at the top right of the Edit page allow you to [change the user's password](</docs/portal/registry-users/changing-passwords.md>), [delete the user](</docs/portal/registry-users/deleting-users.md>), and [update the user](</docs/portal/registry-users/updating-users.md>).
```

Comment:
Please change "to edit the repositories" to "to view the projects". Note that the user is associated with projects rather than repositories. And "edit" shall be replaced with "view" for reason stated in above comments.
Please change "the Edit page" to "the detail page" as well.

Contents on this page need improvement for better consistency with previous pages. Please refer to my comment on the "Editing Registry Projects" page.

Reply:
This topic was deprecated as a result of my reorganizing the registry project topics to make them conform to the structure of previous topics.